### ISSUE TITLE
JAWS2020 VERSION NOT ANNOUNCING LIVE REGIONS 

### ISSUE THREAD

**Issue by SJALADHANKI** [FreedomScientific/VFO-standards-support#613]
JAWS fails to announce the Live region when using the following combinations - Electron Chrome 66 browser + JAWS2018, JAWS2019 JAWS2020, JAWS2021, JAWS2022. However, JAWS18, NVDA 2021, VoiceOver announce live region successfully. 

Browser: Chrome version 66.0.3359.181, Electron version 3.0.0. 

### Expected result
> JAWS 2018, 2019, 2020, 2021 and 2022 must announce the live region. 

### Actual result
> JAWS 2018, 2019, 2020, 2021 and 2022 do not announce the live region. 

### Example
> A test case that demonstrates the issue -
On clicking the check button, the contents of the live region change to the current time. However, JAWS 2020 does not announce the live region. 

`<div role="region" aria-live="assertive" aria-atomic="false" aria-relevant="text" id="clock"> What is the time?</div>`
`<button type="button" onclick='focusFunction()'>Check</button>`
`<button type="button" onclick='focusReset()'>Reset</button>
`
```
<script>
	const d = new Date();
	let time = d.getHours();
	let time1 = d.getMinutes();
	let time2 = d.getSeconds();
	var btn = document.getElementById("clock");
	function focusFunction() {		
		btn.innerHTML = time + ":" + time1 + ":" + time2;
	}
	function focusReset() {		
		btn.innerHTML = 'What is the time?';
	}	
</script>
```

**Comment by sivakusayan** [comment]
This seems like a duplicate of this issue: https://github.com/FreedomScientific/VFO-standards-support/issues/602

**Comment by sivakusayan** [comment]
FYI, the Freedom Scientific development team reached out to me and gave a possible solution, so issue #602 is now closed. You should see if their suggested solution works for you too.

**Comment by SJALADHANKI** [comment]
The Freedom Scientific development team reached out to us and provided a possible solution. The solution was reviewed with JAWS 2020 and JAWS 2022, it worked fine. I will mark the issue #613 as closed. 

Solution: 
> Part 1 - Code change: Use aria-atomic = "true"
> Part 2 - Config change: A quote from the Freedom Scientific development team:
Live regions are enabled by default for Google Chrome, Microsoft Edge and Mozilla Firefox. Live regions are turned off by default for all other apps due to their misuse and excessive nature. If other applications are compiled using the electron framework with a different name, than the standard browsers, you can turn on live region announcements. Currently it seems you can’t do it in the UI however.
To enable live regions for an application, or even in default, load or create a jcf file with the following key set to 1 in the [HTML] section:
e.g.
[HTML]
AnnounceLiveRegionUpdates=1

If you have a script for your program you can find the JCF file in:
C:\Users\username\AppData\Roaming\Freedom Scientific\JAWS\2022\Settings\enu
